govcd/edgegateway.go

```diff
@@ -177,14 +177,21 @@ func (eGW *EdgeGateway) RemoveNATPortMapping(natType, externalIP, externalPort s
 
 }
 
+func (eGW *EdgeGateway) AddNetworkNATMapping(network *types.OrgVDCNetwork, natType, externalIP, internalIP string) (Task, error) {
+	return eGW.AddNATPortMappingWithUplink(network, natType, externalIP, "any", internalIP, "any", "any", "")
```

Comment:
I'm a bit confused, why don't we just use the existing `AddNATPortMappingWithUplink` as opposed to defining the new `AddNetworkNATMapping`?

Reply:
Idea of wrapper is to have simple function against full bloated - as it hides - user don't need what to send to values which you don't use : (network, natType, externalIP, `"any"`, internalIP, `"any"`, `"any"`, `""`)

Reply:
Great (and my bad - on a hurry missed that this wrapper has a lot less params).

Last comment: what about renaming to ‘AddNATWithNetwork’, so we have all public NAT functions with same ‘AddNAT...’ beginning?

Reply:
Well I don't like notation `withNetwork`  cause by default it is bug without network.
I changed to: `AddNATRule` - I hope it's better

Reply:
It’s great! But the comments still point to old name ;)

Reply:
Yep, fixed:)